website/src/install.md

```diff
@@ -23,10 +23,11 @@
 Cargo 1.75.0 or later is required to build.
 </div>
 
-Add `iceberg` into `Cargo.toml` dependencies:
+Add `iceberg` and `iceberg-catalog-rest` into `Cargo.toml` dependencies:
```

Comment:
How about splitting them into two sections as following:
```
Add `iceberg-catalog-rest` if you want to use rest catalog.

...
```

Reply:
Makes sense. 
I'll push these changes in a day or two